### PR TITLE
Milestone 1 - Mobile menu (Re-review)

### PR DESCRIPTION
**Pull request by mukhammadrizooff**
# Portfolio: Mobile menu

**For this project, I add the JavaScript + HTML + CSS linter.**

## I did the following on my mobile-menu:

- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+function main() {
+  const menu = document.querySelector('#menu-container');
+  const hamburger = document.querySelector('#humburger-btn');
+  const body = document.querySelector('body');
+  const anchorTags = document.querySelectorAll('.mobile-menu-ul li');
+
```
Comment (cwaku) [inline script.js:6]: You seem to be missing the close button. I think this line of code should be of help to close the mobile menu
`const closeButton = document.querySelector('.close-btn');`

File: script.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  }
+
+  function closeMenuIcon() {
+    menu.classList.remove('visible');
+    body.classList.remove('overflow-hidden');
+  }
+
+  hamburger.addEventListener('clicked', openMenuIcon);
+  anchorTags.forEach((link) => {
+    link.addEventListener('clicked', closeMenuIcon);
```
Comment (cwaku) [inline script.js:19]: Your menu button isn't working at the moment. Have a look at the spelling of the event here.
It is should be `click` and not `clicked`. Please change that and everything should work fine

**Review by cwaku — CHANGES_REQUESTED** [review]
Hi @mukhammadrizooff, 

Great job so far!

- [x] All linters passed 👍🏾 
- [x] Correct use of Gitflow 💯 
- [x] Repo looks professional 👍🏾 
- [x] Descriptive commit and PR messages 🥇 



You still need to work on some issues to prepare your project for the final evaluation, but don't give up you are almost there!


## Suggested changes
_Check the comments under the review._

_You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by Sahar-AbdelSamad — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hi @mukhammadrizooff :wave: :wave: , 

Great work on making the changes requested by the previous reviewer 👏  :smile: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

:heavy_check_mark: The hamburger button works perfectly
:heavy_check_mark: Your project look awesome

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**